python/spear/env.py

```diff
@@ -77,13 +87,13 @@ def close(self):
 
     def begin_tick(self):
         self._instance.engine_service.begin_tick()
-        self._instance.game_world_service.set_game_paused(False)
+        self._instance.game_world_service.call_function(uobject=self.gameplay_statics_default_object, ufunction=self.set_game_paused_func, args={"WorldContextObject": self.gameplay_statics_default_object, "bPaused": False})
```

Comment:
The Blueprint system handles `WorldContextObject` args automatically and we should too. See comment above.

```
self._instance.game_world_service.call_function(uobject=self.gameplay_statics_default_object, ufunction=self.set_game_paused_func, args={"bPaused": False})
```